apps/frontend/web/src/pages/Login.tsx

```diff
@@ -1,5 +1,107 @@
-const Login = () => {
-  return <div>{'Login'}</div>;
-};
+import { ArrowRightIcon, ViewIcon, ViewOffIcon } from '@chakra-ui/icons';
+import {
+    Box,
+    Button,
+    Center,
+    Divider,
+    FormControl,
+    FormLabel,
+    Heading,
+    Input,
+    InputGroup,
+    InputRightElement,
+    Link,
+    Text,
+    VStack,
+  } from '@chakra-ui/react';
+  import { useState } from 'react';
+  
+  function Login() {
+    const [show, setShow] = useState(false);
```

Comment:
Mettre un préfixe `should`, `is`, ou `must` devant les variables de booléens

Reply:
il faut voir avec segun car cela vient de la doc chakra